src/Symfony/Component/Console/Helper/SymfonyQuestionHelper.php

```diff
@@ -97,14 +97,14 @@ protected function writePrompt(OutputInterface $output, Question $question)
         $output->writeln($text);
 
         if ($question instanceof ChoiceQuestion) {
-            $width = max(array_map('strlen', array_keys($question->getChoices())));
+            $messages = $this->prepareChoices('comment', $question);
 
-            foreach ($question->getChoices() as $key => $value) {
-                $output->writeln(sprintf("  [<comment>%-${width}s</comment>] %s", $key, $value));
-            }
+            $output->writeln($messages);
+            // ChoiceQuestion can have any prompt
+            $output->write($question->getPrompt() ?: Question::DEFAULT_PROMPT);
```

Comment:
`getPrompt()` will always return a string. The constant is also useless. Just write the prompt.

Reply:
That's another bug fixed by this PR btw. Unrelated to the linked issue.